test/auth.js

```diff
@@ -0,0 +1,22 @@
+export const auth = {
+  strategy: 'jwt_token',
+  credentials: {
+    user: {
```

Comment:
Do we definitely need to stub the entire `credentials.user` for `server.inject()`?

Only mentioning this because we paired it with `auth.access.entity` in **forms-designer**

```patch
auth: {
- mode: 'required'
+ mode: 'required',
+ access: {
+   entity: 'user'
+ }
}
```